libs/npm/core/src/components/table/FlexboxTable.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+
+import { Text } from '../form';
+import { ITableProps, SortFlag, TablePager, useTable } from '.';
+import * as styled from './styled';
+
+export const FlexboxTable = <T extends object>({
```

Comment:
This is a default implementation.  We can create other implementations because it's headless.